header-footer-grid/Core/Magic_Tags.php

```diff
@@ -290,6 +290,16 @@ public function site_tagline() {
 		return get_bloginfo( 'description' );
 	}
 
+	/**
+	 * Title for the page currently being viewed.
+	 *
+	 * @return string
+	 */
+	public function current_query_title() {
+		$show_homepage_name = apply_filters( 'neve_query_title_magic_tag_on_homepage', false );
+		return ( $show_homepage_name && is_front_page() ) ? get_bloginfo( 'name' ) : wp_title( '' );
```

Comment:
If I have a static page set as the front page and the page title is "Home", the return value will be the name of the blog instead of the name of the page